Sources/MapboxMaps/Annotations/Generated/PointAnnotationManager.swift

```diff
@@ -317,10 +317,10 @@ public class PointAnnotationManager: AnnotationManagerInternal {
     /// Font stack to use for displaying text.
     public var textFont: [String]? {
         get {
-            return layerProperties["text-font"] as? [String]
+            return (layerProperties["text-font"] as? [Any])?[1] as? [String]
```

Comment:
Is it safe to assume that the array contains at least 2 elements?

Reply:
Yes. The value for this key is only set in the corresponding setter to either nil or a length-2 array.